libs/design/src/atoms/form/form-field/form-field.component.scss

```diff
@@ -0,0 +1,10 @@
+.daff-form-field {
+  display: block;
+  padding-bottom: 24px;
+  position: relative;
+
+  &__error-message {
```

Comment:
@lderrickable after reviewing this with @xelaint I think we've decided that it's ok for the content to "jump" if a field goes from "indeterminate" to "invalid".

Reply:
removed padding and position styling for `daff-form-field`